MAINTAINERS.md

```diff
@@ -0,0 +1,25 @@
+# Istio Maintainers
```

Comment:
are we changing istio WG leads to maintainers?

Reply:
No, just renaming "Approvers" to "Maintainers".